phpunit.xml.dist

```diff
@@ -30,4 +30,8 @@
         </whitelist>
     </filter>
 
+    <php>
+        <ini name="precision" value="8"/>
+    </php>
+
```

Comment:
Can you please show me where the sonata admin uses the precison value? Haven't found any location where we rely on this value.